README.md

```diff
@@ -1 +1,54 @@
-# designing_microservices_course
\ No newline at end of file
+# Designing microservices course
+This repository is intended for demonstration purposes for students during the "Basics of microservices architecture 
+course" which is going to be taught in 2019 autumn term for 4 years students if ITMO university, department of 
+informational systems.
+
+## Disclaimer
+Project is created to demonstrate main approaches and some of patterns which are used for creating microservices-based
+system. Some of architectural decisions were made especially for complicating system in order for us to have a room
+for applying some of patters, some were made to simplify system to order for us not having to implement real-world 
+business logic. Some of the services or business logic are just a mocks. We also didn't aim to create perfect code
```

Comment:
We aimed to create a decent example with somewhat clear, tested code with logging and without duplicates. However it cannot be considered as a source of best practices in microservices design.